### ISSUE TITLE
[Developer] Context Help area is missing content and has broken links

### ISSUE THREAD

**Issue by mcdurdin** [keymanapp/keyman#2130]
**Describe the bug**
The Context Help area in Keyman Developer has many missing topics and links that don't work. It has not kept up in recent versions.

Issues include:
- [x] Broken 'more help' link
- [x] Clicking in the help form tries to show help for the form itself
- [x] Keyman Developer used to keep track of which help topics were missing in `%appdata%\Keyman\Keyman Developer\helpmissing.xml`. This is stubbed out in 12.0.

* Note: missing topics and missing context paths are documented in #2131.
---

**Keyman Developer**:
 - OS: Windows 10
 - Keyman Developer version: 12.0

**Comment by mcdurdin** [comment]
Missing help topics, generated from a quick run through of Keyman Developer UI (may not be exhaustive), mined from helpmissing.xml:

```xml
<?xml version="1.0"?>
<MissingTopics>
  <MissingTopic FormName="context/keyboard-editor" ControlName="editKeyboardVersion" LastVisited="25/09/2019 6:54:23 AM" VisitCount="6"/>
  <MissingTopic FormName="context/startup" ControlName="" LastVisited="25/09/2019 6:52:40 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-identifier" ControlName="sgChars" LastVisited="25/09/2019 6:57:48 AM" VisitCount="2"/>
  <MissingTopic FormName="" ControlName="cefwp" LastVisited="25/09/2019 6:52:46 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/store" ControlName="cefwp" LastVisited="25/09/2019 6:59:04 AM" VisitCount="18"/>
  <MissingTopic FormName="language/reference/name" ControlName="cefwp" LastVisited="25/09/2019 6:54:14 AM" VisitCount="3"/>
  <MissingTopic FormName="language/reference/version" ControlName="cefwp" LastVisited="25/09/2019 7:00:11 AM" VisitCount="14"/>
  <MissingTopic FormName="language/reference/begin" ControlName="cefwp" LastVisited="25/09/2019 6:54:15 AM" VisitCount="2"/>
  <MissingTopic FormName="language/reference/use" ControlName="cefwp" LastVisited="25/09/2019 6:53:00 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/outs" ControlName="cefwp" LastVisited="25/09/2019 6:53:10 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/any" ControlName="cefwp" LastVisited="25/09/2019 6:54:06 AM" VisitCount="47"/>
  <MissingTopic FormName="language/reference/index" ControlName="cefwp" LastVisited="25/09/2019 6:53:18 AM" VisitCount="1"/>
  <MissingTopic FormName="language/guide/comments" ControlName="cefwp" LastVisited="25/09/2019 6:59:05 AM" VisitCount="56"/>
  <MissingTopic FormName="language/reference/group" ControlName="cefwp" LastVisited="25/09/2019 6:53:27 AM" VisitCount="6"/>
  <MissingTopic FormName="language/reference/context" ControlName="cefwp" LastVisited="25/09/2019 6:53:36 AM" VisitCount="9"/>
  <MissingTopic FormName="language/reference/beep" ControlName="cefwp" LastVisited="25/09/2019 6:53:36 AM" VisitCount="6"/>
  <MissingTopic FormName="language/reference/platform" ControlName="cefwp" LastVisited="25/09/2019 6:53:48 AM" VisitCount="7"/>
  <MissingTopic FormName="language/reference/nul" ControlName="cefwp" LastVisited="25/09/2019 6:53:49 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/copyright" ControlName="cefwp" LastVisited="25/09/2019 6:54:14 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/message" ControlName="cefwp" LastVisited="25/09/2019 6:54:14 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/targets" ControlName="cefwp" LastVisited="25/09/2019 6:54:14 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/layoutfile" ControlName="cefwp" LastVisited="25/09/2019 6:54:14 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/keyboardversion" ControlName="cefwp" LastVisited="25/09/2019 6:54:14 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/bitmap" ControlName="cefwp" LastVisited="25/09/2019 6:54:15 AM" VisitCount="1"/>
  <MissingTopic FormName="language/reference/visualkeyboard" ControlName="cefwp" LastVisited="25/09/2019 6:54:15 AM" VisitCount="1"/>
  <MissingTopic FormName="context/keyboard-editor" ControlName="gridFeatures" LastVisited="25/09/2019 6:54:23 AM" VisitCount="4"/>
  <MissingTopic FormName="context/keyboard-editor" ControlName="cmdAddFeature" LastVisited="25/09/2019 6:54:23 AM" VisitCount="4"/>
  <MissingTopic FormName="context/keyboard-editor" ControlName="cmdEditFeature" LastVisited="25/09/2019 6:54:23 AM" VisitCount="4"/>
  <MissingTopic FormName="context/keyboard-editor" ControlName="cmdRemoveFeature" LastVisited="25/09/2019 6:54:23 AM" VisitCount="4"/>
  <MissingTopic FormName="context/keyboard-editor" ControlName="pages" LastVisited="25/09/2019 6:54:43 AM" VisitCount="14"/>
  <MissingTopic FormName="context/keyboard-editor" ControlName="palColours" LastVisited="25/09/2019 6:54:27 AM" VisitCount="4"/>
  <MissingTopic FormName="context/keyboard-editor#toc-on-screen-tab" ControlName="pages" LastVisited="25/09/2019 6:54:30 AM" VisitCount="3"/>
  <MissingTopic FormName="context/keyboard-editor#toc-on-screen-tab" ControlName="chkFillUnderlyingLayout" LastVisited="25/09/2019 6:54:30 AM" VisitCount="3"/>
  <MissingTopic FormName="context/keyboard-editor" ControlName="pagesTouchLayout" LastVisited="25/09/2019 6:54:31 AM" VisitCount="1"/>
  <MissingTopic FormName="context/keyboard-editor#toc-touch-layout-tab" ControlName="cefwp" LastVisited="25/09/2019 6:54:31 AM" VisitCount="1"/>
  <MissingTopic FormName="context/editor" ControlName="cefwp" LastVisited="25/09/2019 6:57:32 AM" VisitCount="4"/>
  <MissingTopic FormName="context/keyboard-editor" ControlName="cmdPackageForDistribution" LastVisited="25/09/2019 6:54:43 AM" VisitCount="5"/>
  <MissingTopic FormName="" ControlName="pages" LastVisited="25/09/2019 6:59:07 AM" VisitCount="5"/>
  <MissingTopic FormName="" ControlName="" LastVisited="25/09/2019 6:54:47 AM" VisitCount="1"/>
  <MissingTopic FormName="context/package-editor" ControlName="pages" LastVisited="25/09/2019 6:56:03 AM" VisitCount="38"/>
  <MissingTopic FormName="context/package-editor" ControlName="lbKeyboards" LastVisited="25/09/2019 6:55:01 AM" VisitCount="5"/>
  <MissingTopic FormName="context/package-editor" ControlName="cbKeyboardOSKFont" LastVisited="25/09/2019 6:54:56 AM" VisitCount="4"/>
  <MissingTopic FormName="context/package-editor" ControlName="cbKeyboardDisplayFont" LastVisited="25/09/2019 6:54:56 AM" VisitCount="4"/>
  <MissingTopic FormName="context/package-editor" ControlName="gridKeyboardLanguages" LastVisited="25/09/2019 6:54:56 AM" VisitCount="4"/>
  <MissingTopic FormName="context/package-editor" ControlName="cmdKeyboardAddLanguage" LastVisited="25/09/2019 6:54:56 AM" VisitCount="4"/>
  <MissingTopic FormName="context/package-editor" ControlName="cmdKeyboardRemoveLanguage" LastVisited="25/09/2019 6:54:56 AM" VisitCount="4"/>
  <MissingTopic FormName="context/package-editor" ControlName="cmdKeyboardEditLanguage" LastVisited="25/09/2019 6:54:56 AM" VisitCount="4"/>
  <MissingTopic FormName="context/package-editor" ControlName="lbLexicalModels" LastVisited="25/09/2019 6:55:30 AM" VisitCount="22"/>
  <MissingTopic FormName="context/package-editor" ControlName="cmdLexicalModelLanguageAdd" LastVisited="25/09/2019 6:55:30 AM" VisitCount="10"/>
  <MissingTopic FormName="context/package-editor" ControlName="chkLexicalModelRTL" LastVisited="25/09/2019 6:55:31 AM" VisitCount="11"/>
  <MissingTopic FormName="context/package-editor" ControlName="editLexicalModelDescription" LastVisited="25/09/2019 6:55:26 AM" VisitCount="1"/>
  <MissingTopic FormName="context/package-editor" ControlName="editLexicalModelVersion" LastVisited="25/09/2019 6:55:26 AM" VisitCount="1"/>
  <MissingTopic FormName="context/package-editor" ControlName="chkFollowKeyboardVersion" LastVisited="25/09/2019 6:55:40 AM" VisitCount="5"/>
  <MissingTopic FormName="context/package-editor" ControlName="lbStartMenuEntries" LastVisited="25/09/2019 6:55:46 AM" VisitCount="3"/>
  <MissingTopic FormName="context/package-editor" ControlName="cmdStartTestOnline" LastVisited="25/09/2019 6:56:03 AM" VisitCount="8"/>
  <MissingTopic FormName="context/package-editor" ControlName="lbDebugHosts" LastVisited="25/09/2019 6:56:03 AM" VisitCount="8"/>
  <MissingTopic FormName="context/package-editor" ControlName="cmdOpenDebugHost" LastVisited="25/09/2019 6:56:03 AM" VisitCount="3"/>
  <MissingTopic FormName="context/package-editor" ControlName="editBootstrapMSI" LastVisited="25/09/2019 6:56:04 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-project" ControlName="lvItems" LastVisited="25/09/2019 6:56:49 AM" VisitCount="6"/>
  <MissingTopic FormName="context/new-project" ControlName="cmdOK" LastVisited="25/09/2019 6:56:51 AM" VisitCount="6"/>
  <MissingTopic FormName="context/new-project" ControlName="cmdCancel" LastVisited="25/09/2019 6:56:23 AM" VisitCount="3"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="editKeyboardName" LastVisited="25/09/2019 6:56:38 AM" VisitCount="4"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="editCopyright" LastVisited="25/09/2019 6:56:31 AM" VisitCount="3"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="editVersion" LastVisited="25/09/2019 6:56:31 AM" VisitCount="3"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="editAuthor" LastVisited="25/09/2019 6:56:30 AM" VisitCount="2"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="clbTargets" LastVisited="25/09/2019 6:56:31 AM" VisitCount="2"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="cmdKeyboardAddLanguage" LastVisited="25/09/2019 6:56:42 AM" VisitCount="5"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="editPath" LastVisited="25/09/2019 6:56:37 AM" VisitCount="3"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="cmdBrowse" LastVisited="25/09/2019 6:56:45 AM" VisitCount="4"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="editKeyboardID" LastVisited="25/09/2019 6:56:37 AM" VisitCount="3"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="cmdCancel" LastVisited="25/09/2019 6:56:46 AM" VisitCount="4"/>
  <MissingTopic FormName="context/select-bcp47-language" ControlName="cbLanguageTag" LastVisited="25/09/2019 6:56:55 AM" VisitCount="2"/>
  <MissingTopic FormName="context/select-bcp47-language" ControlName="cbScriptTag" LastVisited="25/09/2019 6:56:34 AM" VisitCount="1"/>
  <MissingTopic FormName="context/select-bcp47-language" ControlName="cbRegionTag" LastVisited="25/09/2019 6:56:34 AM" VisitCount="1"/>
  <MissingTopic FormName="context/select-bcp47-language" ControlName="editLanguageName" LastVisited="25/09/2019 6:56:34 AM" VisitCount="1"/>
  <MissingTopic FormName="context/select-bcp47-language" ControlName="cmdResetLanguageName" LastVisited="25/09/2019 6:56:34 AM" VisitCount="1"/>
  <MissingTopic FormName="context/select-bcp47-language" ControlName="cmdOK" LastVisited="25/09/2019 6:56:57 AM" VisitCount="2"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="cmdKeyboardEditLanguage" LastVisited="25/09/2019 6:56:43 AM" VisitCount="2"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="cmdKeyboardRemoveLanguage" LastVisited="25/09/2019 6:56:43 AM" VisitCount="2"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="editProjectFilename" LastVisited="25/09/2019 6:56:39 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-project-parameters" ControlName="" LastVisited="25/09/2019 6:56:44 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="editAuthor" LastVisited="25/09/2019 6:57:02 AM" VisitCount="3"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="editModelName" LastVisited="25/09/2019 6:57:03 AM" VisitCount="4"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="editCopyright" LastVisited="25/09/2019 6:57:02 AM" VisitCount="3"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="editVersion" LastVisited="25/09/2019 6:57:01 AM" VisitCount="2"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="cmdAddLanguage" LastVisited="25/09/2019 6:56:57 AM" VisitCount="2"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="cmdEditLanguage" LastVisited="25/09/2019 6:56:58 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="cmdRemoveLanguage" LastVisited="25/09/2019 6:56:58 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="editPath" LastVisited="25/09/2019 6:56:58 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="cmdBrowse" LastVisited="25/09/2019 6:56:59 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="editAuthorID" LastVisited="25/09/2019 6:56:59 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="cbBCP47" LastVisited="25/09/2019 6:56:59 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="editUniq" LastVisited="25/09/2019 6:56:59 AM" VisitCount="1"/>
  <MissingTopic FormName="context/new-model-project-parameters" ControlName="cmdCancel" LastVisited="25/09/2019 6:57:00 AM" VisitCount="1"/>
  <MissingTopic FormName="context/wordlist-editor" ControlName="pages" LastVisited="25/09/2019 6:57:21 AM" VisitCount="1"/>
  <MissingTopic FormName="context/wordlist-editor" ControlName="gridWordlist" LastVisited="25/09/2019 6:57:23 AM" VisitCount="1"/>
  <MissingTopic FormName="context/wordlist-editor" ControlName="cmdDeleteRow" LastVisited="25/09/2019 6:57:27 AM" VisitCount="1"/>
  <MissingTopic FormName="context/wordlist-editor" ControlName="cmdSortByFrequency" LastVisited="25/09/2019 6:57:28 AM" VisitCount="1"/>
  <MissingTopic FormName="context/editor" ControlName="" LastVisited="25/09/2019 6:57:31 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-identifier" ControlName="gridFonts" LastVisited="25/09/2019 6:57:50 AM" VisitCount="2"/>
  <MissingTopic FormName="context/messages" ControlName="memoMessage" LastVisited="25/09/2019 6:57:51 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdFilter" LastVisited="25/09/2019 6:58:06 AM" VisitCount="2"/>
  <MissingTopic FormName="context/character-map" ControlName="rbRange" LastVisited="25/09/2019 6:57:59 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="editRangeStart" LastVisited="25/09/2019 6:58:00 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="rbName" LastVisited="25/09/2019 6:58:00 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdCharactersInCurrentFontOnly" LastVisited="25/09/2019 6:58:02 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdBlock" LastVisited="25/09/2019 6:58:02 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdStar" LastVisited="25/09/2019 6:58:03 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdQuestionMark" LastVisited="25/09/2019 6:58:03 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdRange" LastVisited="25/09/2019 6:58:03 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdSet" LastVisited="25/09/2019 6:58:03 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdDollar" LastVisited="25/09/2019 6:58:03 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdOK" LastVisited="25/09/2019 6:58:04 AM" VisitCount="1"/>
  <MissingTopic FormName="context/character-map" ControlName="cmdCancel" LastVisited="25/09/2019 6:58:04 AM" VisitCount="1"/>
  <MissingTopic FormName="context/debug" ControlName="memo" LastVisited="25/09/2019 6:58:59 AM" VisitCount="12"/>
  <MissingTopic FormName="context/debug#toc-regression-testing" ControlName="lbRegTestLog" LastVisited="25/09/2019 6:58:39 AM" VisitCount="2"/>
  <MissingTopic FormName="context/debug" ControlName="sgChars" LastVisited="25/09/2019 6:58:29 AM" VisitCount="1"/>
  <MissingTopic FormName="context/debug#toc-state" ControlName="cmdRestartDebugger" LastVisited="25/09/2019 6:58:33 AM" VisitCount="1"/>
  <MissingTopic FormName="context/debug#toc-state" ControlName="lbKeystrokeLog" LastVisited="25/09/2019 6:58:33 AM" VisitCount="1"/>
  <MissingTopic FormName="context/debug#toc-elements" ControlName="lvElements" LastVisited="25/09/2019 6:58:34 AM" VisitCount="1"/>
  <MissingTopic FormName="context/debug#toc-call-stack" ControlName="lbCallStack" LastVisited="25/09/2019 6:58:35 AM" VisitCount="1"/>
  <MissingTopic FormName="context/debug#toc-deadkeys" ControlName="lbDeadkeys" LastVisited="25/09/2019 6:58:35 AM" VisitCount="1"/>
  <MissingTopic FormName="context/debug#toc-regression-testing" ControlName="cmdRegTestStartStopLog" LastVisited="25/09/2019 6:58:36 AM" VisitCount="1"/>
  <MissingTopic FormName="context/debug#toc-regression-testing" ControlName="cmdRegTestOptions" LastVisited="25/09/2019 6:58:37 AM" VisitCount="1"/>
  <MissingTopic FormName="context/about-tike" ControlName="cmdOK" LastVisited="25/09/2019 6:59:21 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="pages" LastVisited="25/09/2019 6:59:37 AM" VisitCount="7"/>
  <MissingTopic FormName="context/options" ControlName="chkShowStartupDialog" LastVisited="25/09/2019 6:59:32 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkAllowMultipleInstances" LastVisited="25/09/2019 6:59:32 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkOpenKeyboardFilesInSourceView" LastVisited="25/09/2019 6:59:32 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="editExternalEditorPath" LastVisited="25/09/2019 6:59:32 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="cmdBrowseExternalEditor" LastVisited="25/09/2019 6:59:32 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="cmdResetToolWindows" LastVisited="25/09/2019 6:59:31 AM" VisitCount="1"/>
  <MissingTopic FormName="context/options" ControlName="cmdSMTPSettings" LastVisited="25/09/2019 6:59:31 AM" VisitCount="1"/>
  <MissingTopic FormName="context/options" ControlName="cmdProxySettings" LastVisited="25/09/2019 6:59:32 AM" VisitCount="1"/>
  <MissingTopic FormName="context/options" ControlName="cmdCancel" LastVisited="25/09/2019 6:59:37 AM" VisitCount="7"/>
  <MissingTopic FormName="context/options" ControlName="cmdOK" LastVisited="25/09/2019 6:59:37 AM" VisitCount="6"/>
  <MissingTopic FormName="context/options" ControlName="chkUseTab" LastVisited="25/09/2019 6:59:33 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="editIndentSize" LastVisited="25/09/2019 6:59:33 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkLinkFontSizes" LastVisited="25/09/2019 6:59:33 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="cmdDefaultFont" LastVisited="25/09/2019 6:59:33 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="cmdQuotedFont" LastVisited="25/09/2019 6:59:33 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="cbEditorTheme" LastVisited="25/09/2019 6:59:33 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkUseOldDebugger" LastVisited="25/09/2019 6:59:35 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkDebuggerBreakWhenExitingLine" LastVisited="25/09/2019 6:59:35 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkDebuggerSingleStepAfterBreak" LastVisited="25/09/2019 6:59:35 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkDebuggerShowStoreOffset" LastVisited="25/09/2019 6:59:35 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkDebuggerAutoRecompile" LastVisited="25/09/2019 6:59:35 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="editWebHostDefaultPort" LastVisited="25/09/2019 6:59:35 AM" VisitCount="2"/>
  <MissingTopic FormName="context/options" ControlName="chkCharMapAutoLookup" LastVisited="25/09/2019 6:59:37 AM" VisitCount="3"/>
  <MissingTopic FormName="context/options" ControlName="cmdCharMapRebuildDatabase" LastVisited="25/09/2019 6:59:37 AM" VisitCount="3"/>
  <MissingTopic FormName="context/options" ControlName="editDatabasePath" LastVisited="25/09/2019 6:59:37 AM" VisitCount="3"/>
  <MissingTopic FormName="context/key-test" ControlName="cmdInsert" LastVisited="25/09/2019 6:59:42 AM" VisitCount="2"/>
  <MissingTopic FormName="context/key-test" ControlName="chkLRDistinguish" LastVisited="25/09/2019 6:59:41 AM" VisitCount="1"/>
  <MissingTopic FormName="context/key-test" ControlName="cmdClose" LastVisited="25/09/2019 6:59:43 AM" VisitCount="1"/>
  <MissingTopic FormName="context/keyboard-fonts" ControlName="fcbCode" LastVisited="25/09/2019 6:59:55 AM" VisitCount="2"/>
  <MissingTopic FormName="context/keyboard-fonts" ControlName="editCodeSize" LastVisited="25/09/2019 6:59:54 AM" VisitCount="1"/>
  <MissingTopic FormName="context/keyboard-fonts" ControlName="fcbChar" LastVisited="25/09/2019 6:59:55 AM" VisitCount="1"/>
  <MissingTopic FormName="context/keyboard-fonts" ControlName="fcbOSK" LastVisited="25/09/2019 6:59:55 AM" VisitCount="1"/>
  <MissingTopic FormName="context/keyboard-fonts" ControlName="editOSKSize" LastVisited="25/09/2019 6:59:55 AM" VisitCount="1"/>
  <MissingTopic FormName="context/keyboard-fonts" ControlName="cmdCancel" LastVisited="25/09/2019 6:59:55 AM" VisitCount="1"/>
  <MissingTopic FormName="" ControlName="editOutputPath" LastVisited="25/09/2019 7:00:07 AM" VisitCount="2"/>
  <MissingTopic FormName="" ControlName="chkCompilerWarningsAsErrors" LastVisited="25/09/2019 7:00:08 AM" VisitCount="2"/>
  <MissingTopic FormName="" ControlName="chkCheckFilenameConventions" LastVisited="25/09/2019 7:00:09 AM" VisitCount="2"/>
  <MissingTopic FormName="" ControlName="cmdOK" LastVisited="25/09/2019 7:00:10 AM" VisitCount="2"/>
  <MissingTopic FormName="" ControlName="chkWarnDeprecatedCode" LastVisited="25/09/2019 7:00:09 AM" VisitCount="1"/>
  <MissingTopic FormName="" ControlName="cmdCancel" LastVisited="25/09/2019 7:00:10 AM" VisitCount="1"/>
</MissingTopics>
```

**Comment by mcdurdin** [comment]
Resolved in 12.0.37
